ietf/meeting/views.py

```diff
@@ -1691,7 +1690,7 @@ def api_get_agenda_data (request, num=None):
         },
         "categories": filter_organizer.get_filter_categories(),
         "isCurrentMeeting": is_current_meeting,
-        "useNotes": meeting.uses_notes(),
+        "uses_notes": meeting.uses_notes(),
```

Comment:
Should also stay as a camelCase key (`usesNotes`) to match the rest of the structure